src/configs/strings/en.json

```diff
@@ -947,6 +947,22 @@
     "message": "Age",
     "description": "the length of time that a person has lived or a thing has existed"
   },
+  "STUDENTS_TITLE": {
+    "message": "Students",
+    "description": "title: the children receiving MDA"
+  },
+  "UPLOADED_STUDENT_LISTS": {
+    "message": "Uploaded Student Lists",
+    "description": "Export Student List"
+  },
+  "EXPORT_STUDENT_LIST": {
+    "message": "Export Student List",
+    "description": "button text: to download a CSV of students per jurisdiction"
+  },
```

Comment:
Are we required to name this "students"?  Isn't it really OpenSRP clients?

Reply:
@moshthepitt the students title are based on the mock in this issue
https://github.com/OpenSRP/opensrp-client-reveal/issues/679

Reply:
Cool, thanks.  The label that is applied here needs to be configurable.  In the case of NTD in Eswatini it is "student" (this will be supplied as an env var) but the default should be "client".